controllers/topics.controllers.js

```diff
@@ -1,8 +1,14 @@
-const { fetchTopics } = require("../models/topics.models");
+const { fetchTopics, fetchArticle } = require("../models/topics.models");
 
 exports.getTopics = (req, res) => {
   fetchTopics().then((topics) => {
-      console.log(topics)
     res.status(200).send({ topics });
   });
 };
+exports.getArticle = (req, res) => {
+  const id = parseInt(req.params.article_id);
```

Comment:
You shouldn't need to parse this - and you could do some de-structuring to make that line even shorter, with something like `const {article_id} = req.params`